### PR TITLE
add reproduction for bug when partial update of both map and ngram fi..

### PR DESCRIPTION
**Pull request by arnej27959**
…eld happens

@geirst please review and merge
@bratseth @vekterli FYI

### CODE REVIEW

File: tests/search/bugs/map-and-ngram/bug_map_and_ngram.rb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+               search(SearchCluster.new("multitest").
+                      num_parts(1).redundancy(1).ready_copies(1).
+                      sd(selfdir+"foo.sd").
+                      indexing("mycc")))
+    start
+    feed_and_wait_for_docs("foo", 3, :file => selfdir+"feed.xml")
+
+    node = vespa.adminserver
+    node.copy(selfdir + "updates.json", dirs.tmpdir)
+    node.execute("cd #{dirs.tmpdir} && vespa-feeder --trace 4 --maxpending 1 updates.json");
```
Comment (geirst) [inline tests/search/bugs/map-and-ngram/bug_map_and_ngram.rb]: Please use the feed() function instead. Copying files is done automatically. Extra params can be specified as well. See here for which params are supported: https://github.com/vespa-engine/system-test/blob/master/lib/nodetypes/feeder.rb#L147.

File: tests/search/bugs/map-and-ngram/feed.xml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+<document type="foo" id="id:test:foo::1">
```
Comment (geirst) [inline tests/search/bugs/map-and-ngram/feed.xml]: Please use json for new feed files.

File: tests/search/bugs/map-and-ngram/bug_map_and_ngram.rb (hunk 1 above)
Comment (arnej27959) [reply tests/search/bugs/map-and-ngram/bug_map_and_ngram.rb]: now using feed()

File: tests/search/bugs/map-and-ngram/feed.xml (hunk 2 above)
Comment (arnej27959) [reply tests/search/bugs/map-and-ngram/feed.xml]: done

**Comment by arnej27959** [comment]
@geirst please take another look

**Pull request merged**